### DISCUSSION TITLE
conda-forge feedstock created for exactextract

### DISCUSSION THREAD

**Discussion by akrherz** [isciences/exactextract#111] [General]
Greetings.  I've taken an initial swing at a exactextract python package for [conda-forge](https://github.com/conda-forge/exactextract-feedstock).  [cfep-04](https://github.com/conda-forge/cfep/blob/main/cfep-05.md) is currently preventing a simple to install package though due to not having a non-dev release on pypi.  Some comments on this all for feedback here

1) It would be nice to have a 0.2.0 release on pypi, even if not in a fully working state.  You are using a version less than 1, so I am unsure how much stability / lack of bugs one can expect :)
2) If anybody wants to help out with the package maintaining with conda-forge, please let me know.  I just wanted to get the ball rolling.
3) I did not create a `libexactextract` package/artefact yet as I am unsure if there are downstream users of it?

Anyway, I hope this all is helpful and not harmful to this exciting package.

**Comment by dbaston** [comment]
Thanks for this!

Some answers to your questions:

> It would be nice to have a 0.2.0 release on pypi

Hopefully within a couple of weeks. I'm mostly satisfied that the current API is stable enough for people to rely on.

> If anybody wants to help out with the package maintaining with conda-forge,

Please feel free to add me as a co-maintainer, for now.

>  I did not create a libexactextract package/artefact yet as I am unsure if there are downstream users of it?

I would leave it out -- the C++ side of things has been less stable than R/Python, and I think anyone using it in C++ is better off vendoring it.
